Tests/Form/Type/IconSelectorTypeTest.php

```diff
@@ -0,0 +1,633 @@
+                '\f205' => 'fa-toggle-on',
+                '\f152' => 'fa-toggle-right',
+                '\f151' => 'fa-toggle-up',
+                '\f25c' => 'fa-trademark',
+                '\f238' => 'fa-train',
+                '\f224' => 'fa-transgender',
+                '\f225' => 'fa-transgender-alt',
+                '\f1f8' => 'fa-trash',
+                '\f014' => 'fa-trash-o',
+                '\f1bb' => 'fa-tree',
+                '\f181' => 'fa-trello',
+                '\f262' => 'fa-tripadvisor',
+                '\f091' => 'fa-trophy',
+                '\f0d1' => 'fa-truck',
+                '\f1e4' => 'fa-tty',
+                '\f173' => 'fa-tumblr',
+                '\f174' => 'fa-tumblr-square',
+                '\f195' => 'fa-turkish-lira',
+                '\f26c' => 'fa-tv',
+                '\f1e8' => 'fa-twitch',
+                '\f099' => 'fa-twitter',
+                '\f081' => 'fa-twitter-square',
+                '\f0e9' => 'fa-umbrella',
+                '\f0cd' => 'fa-underline',
+                '\f0e2' => 'fa-undo',
+                '\f127' => 'fa-unlink',
+                '\f09c' => 'fa-unlock',
+                '\f13e' => 'fa-unlock-alt',
+                '\f0dc' => 'fa-unsorted',
+                '\f093' => 'fa-upload',
+                '\f155' => 'fa-usd',
+                '\f007' => 'fa-user',
+                '\f0f0' => 'fa-user-md',
+                '\f234' => 'fa-user-plus',
+                '\f21b' => 'fa-user-secret',
+                '\f235' => 'fa-user-times',
+                '\f0c0' => 'fa-users',
+                '\f221' => 'fa-venus',
+                '\f226' => 'fa-venus-double',
+                '\f228' => 'fa-venus-mars',
+                '\f237' => 'fa-viacoin',
+                '\f03d' => 'fa-video-camera',
+                '\f27d' => 'fa-vimeo',
+                '\f194' => 'fa-vimeo-square',
+                '\f1ca' => 'fa-vine',
+                '\f189' => 'fa-vk',
+                '\f027' => 'fa-volume-down',
+                '\f026' => 'fa-volume-off',
+                '\f028' => 'fa-volume-up',
+                '\f071' => 'fa-warning',
+                '\f18a' => 'fa-weibo',
+                '\f1d7' => 'fa-weixin',
+                '\f232' => 'fa-whatsapp',
+                '\f193' => 'fa-wheelchair',
+                '\f1eb' => 'fa-wifi',
+                '\f266' => 'fa-wikipedia-w',
+                '\f17a' => 'fa-windows',
+                '\f159' => 'fa-won',
+                '\f19a' => 'fa-wordpress',
+                '\f0ad' => 'fa-wrench',
+                '\f168' => 'fa-xing',
+                '\f169' => 'fa-xing-square',
+                '\f1d4' => 'fa-y-combinator-square',
+                '\f19e' => 'fa-yahoo',
+                '\f23b' => 'fa-yc',
+                '\f1e9' => 'fa-yelp',
+                '\f157' => 'fa-yen',
+                '\f167' => 'fa-youtube',
+                '\f16a' => 'fa-youtube-play',
+                '\f166' => 'fa-youtube-square',
```

Comment:
I agree. I guess this weird test has been added for color picker to insure the test coverage is not degraded by addition of the Color class without associated test.